File: LiquibaseAdmin/Services/Liquibase/Commands/RollbackCountLiquibaseCommand.cs
using LiquibaseAdmin.Services.Liquibase.Commands.Base;

namespace LiquibaseAdmin.Services.Liquibase.Commands
{
    public class RollbackCountLiquibaseCommand(int count, bool sql = false) : LiquibaseSqlSubversionCommandBase(sql)
    {
        public override string Name => "rollbackCount";

        public override string Params => count.ToString();
    }
}

File: LiquibaseAdminFrontend/src/domain/liquibase/commands/RollbackCountLiquibaseCommand.ts
import { SqlLiquibaseCommandBase } from '@/domain/liquibase/commands/base/SqlLiquibaseCommandBase'

export class RollbackCountLiquibaseCommand extends SqlLiquibaseCommandBase {
    public readonly name = 'rollback-count'

    constructor(
        private readonly count: number,
        sql = true,
    ) {
        super(sql, { count })
    }
}
